src/vstart.sh

```diff
@@ -273,6 +274,7 @@ options:
 	--crimson-smp: number of cores to use for crimson
 	--crimson-alien-num-threads: number of alien-tp threads
 	--crimson-alien-num-cores: number of cores to use for alien-tp
+  --crimson-balance-cpu: distribute the Seastar reactors uniformly across OSDs (osd) or NUMA (socket)s
```

Comment:
Probably good to mention there are totally 3 policies available, IIUC:
* no specify: seastar reactors to use CPUs in order (unbalanced across sockets)
* osd: distribute across sockets uniformly, don't split within an OSD
* socket: distribute across sockets uniformly, split within an OSD

Reply:
Yes, that's correct, as well as the mapping comment above 👍